src/components/reusable/pagetitle/pageTitle.scss

```diff
@@ -0,0 +1,55 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: block;
+}
+
+.page-title {
+  margin: 0;
+  &-wrapper {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0 16px;
+  }
+  &-text-wrapper {
+    word-break: break-word;
+  }
+  &-primary {
+    @include typography.type-headline-04;
+    max-width: 600px;
+  }
+  &-secondary {
+    @include typography.type-headline-06;
+    max-width: 976px;
+  }
+}
+.page {
+  &-headline {
+    @include typography.type-body-01;
+    font-weight: 300;
+    margin-bottom: 4px;
+    max-width: 380px;
```

Comment:
Why the `max-width`s? Unless the design calls for them, just let word wrapping happen naturally.